parasolr/solr/tests/test_client.py

```diff
@@ -38,6 +39,10 @@ def test_as_dict(self):
         # other structures should be untouched
         assert isinstance(as_dict['b']['c'], list)
 
+    def test_repr(self):
+        data = {'a': 1, 'b': 2}
```

Comment:
This is failing on the python 3.5 test because `dict` error is not guaranteed. I think the simplest solution is just to include one item in the `dict`, which should still be sufficient for our purposes.